hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,22 +1,36 @@
-
-Note that 'USB mass storage boot' is different from 'USB device boot mode'. [USB device boot mode](device.md) allows a Raspberry Pi connected to a computer to boot as a USB device, using files from that computer.
-
-If you are unable to use a particular USB device to boot your Raspberry Pi, an alternative is to use the special bootcode.bin-only boot mode as described [here](README.md). This Pi will still boot from the SD card, but `bootcode.bin` is the only file read from it.
+<a name="pi400"></a>
+## Raspberry Pi 400
+To boot the Pi 400 from a USB mass storage device, simply image the USB drive with Raspberry Pi OS 2020-08-20 or newer using the Raspberry Pi Imager utility: select the USB drive from the `SD Card` list in Raspberry Pi Imager.
 
 <a name="pi4"></a>
 ## Raspberry Pi 4
-To enable USB mass storage boot on a Raspberry Pi 4:
+Depending on when your Raspberry Pi 4 was manufactured, the bootloader EEPROM may need to be updated to enable booting from USB mass storage devices. 
+
+### Check if your Pi 4B has the required bootloader EEPROM version
+
+To check if your Pi 4B has the required bootloader EEPROM version, boot the device with no SD card inserted and a display attached to one of the HDMI ports. The Pi 4B will display a diagnostic screen on the attached display, which includes the bootloader EEPROM version at the top of the screen. The bootloader must be dated `Sep 3 2020` or later to support USB mass storage boot. If the diagnostic screen reports a date earlier than `Sep 3 2020`, or there is no diagnostic screen shown, you will need to updated the bootloader EEPROM first to enable USB mass storage boot.
+
+USB mass storage boot on the Pi 4B required Raspberry Pi OS 2020-08-20 or later.
+
 
-* Use the "Misc Utility Images" option in the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to create a SD card with the latest "Raspberry Pi 4 EEPROM boot recovery" image.
-* Update to Raspberry Pi OS 2020-08-20 or newer.
-* Use [raspi-config](../../../configuration/raspi-config.md) to choose between SD/USB (default) or SD/Network boot modes.
+### Enable USB mass storage boot on a Pi 4 by updating the bootloader EEPROM
+If your Pi 4 requires an updated bootloader EEPROM in order to support USB mass storage boot, you can perform the update as follows:
+
+1. Use the "Misc Utility Images" option in [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to create an SD card with the latest "Raspberry Pi 4 EEPROM boot recovery" image.
+1. Boot the Pi 4 using this SD card and a display attached to one of the HDMI ports.
+3. The bootloader EEPROM will be updated to the latest available version, then the display will turn green to indicate success.
+
+The Pi 4 can now be booted from a USB mass storage device.
+
+## Changing boot order on Pi 4 and Pi 400
+
+The [raspi-config](../../../configuration/raspi-config.md) utility can be used to choose between SD/USB (default) or SD/Network boot modes.
```

Comment:
Might be worth adding a brief explanation of what SD/USB and SD/Network actually mean?
(i.e. SD/USB means that the bootloader will attempt USB boot if it can't boot from SD)

Reply:
In the interests of trying to keep things simple, I won't add any extra explanation for that.

Reply:
Fair enough.